examples/call-runtime/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "call-runtime"
+version = "4.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+ink = { path = "../../crates/ink", default-features = false, features = ["call-runtime"] }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
+
+# Substrate
+#
+# We need to explicitly turn off some of the `sp-io` features, to avoid conflicts
+# (especially for global allocator).
+#
+# See also: https://substrate.stackexchange.com/questions/4733/error-when-compiling-a-contract-using-the-xcm-chain-extension.
+sp-io = { version = "12.0.0", default-features = false, features = ["disable_panic_handler", "disable_oom", "disable_allocator"] }
```

Comment:
```suggestion
sp-io = { version = "18.0.0", default-features = false, features = ["disable_panic_handler", "disable_oom", "disable_allocator"] }
```

Reply:
I wouldn't use the latest here. I'd see what `subxt` and `cargo-contract` are using and stick to that

Reply:
yes it does

Reply:
example moved to https://github.com/paritytech/ink-examples/pull/6

Reply:
I suppose it should have been moved to `/integration-tests/` of this repository, instead. Otherwise we don't have it covered by your e2e test. @cmichi please confirm.

Reply:
I must have missed the directory and thought that these contracts are only in the separate repo - sorry for that. The example has been just brought back